backend/store.py

```
@@ -1224,6 +1398,132 @@ def clear_jobs(self):
+        If reading the lock state fails, the lock is removed as a best-effort fallback.
+        
+        Parameters:
+            project_id (str): Identifier of the project whose cluster lock should be released.
+            job_id (str): Identifier of the cluster job expected to own the lock.
+        """
+        key = self._cluster_lock_key(project_id)
+        try:
+            current = self._get(key)
+            if current == job_id:
+                self._delete(key)
+        except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)